operator/olm-base/manifests/clusterserviceversion.yaml

```diff
@@ -1,143 +0,0 @@
-      mediatype: image/png
-  install:
-    spec:
-      deployments:
-        - name: keycloak-operator
-          spec:
-            replicas: 1
-            selector:
-              matchLabels:
-                name: keycloak-operator
-            strategy: {}
-            template:
-              metadata:
-                labels:
-                  name: keycloak-operator
-              spec:
-                containers:
-                  - command:
-                      - java
-                      - -Djava.util.logging.manager=org.jboss.logmanager.LogManager
-                      - -jar
-                      - quarkus-run.jar
-                    env:
-                      - name: QUARKUS_OPERATOR_SDK_NAMESPACES
-                        valueFrom:
-                          fieldRef:
-                            fieldPath: metadata.annotations['olm.targetNamespaces']
```

Comment:
<img width="400" src="https://github.com/keycloak/keycloak/assets/38039883/1e8de0f1-5748-40bc-ba67-8283076720b3"/>

I've noticed the `QUARKUS_OPERATOR_SDK_NAMESPACES` env var was renamed to `KUBERNETES_NAMESPACE`. Can be like that? 

Also the value for the namespace is getting directly from the `metadata.namespace`, which I assume is directly specified in the `template.metadata.namespace` a few lines above and specified as 'placeholder', right? 

Does this [line](https://github.com/keycloak/keycloak/pull/20672/files#diff-f53c77df31edb884b75c19bd62de359748d617a94909dbafece4bbc435c0ee23R18) have any effect on this?

Reply:
@mabartos I read about that on Monday too. Afaik you're correct - we can do it thanks to the values in application.properties file (see [this doc](https://quarkus.io/guides/deploying-to-kubernetes#namespace)).

But let's wait for Alex's confirmation.

Reply:
Latest commit has manual yq commands to fix this